src/test/java/com/wikia/webdriver/testcases/specialpagestests/LicensedVideoSwap.java

```diff
@@ -42,9 +38,9 @@ public void LicensedVideoSwap_002_navigateToHistoryAndBackToLvs() {
     licensedVideoSwap.verifyOnLvsPage();
   }
 
-  @RelatedIssue(issueID = "CONCF-438", comment = "Feature will be deprecated. No need to test manually")
```

Comment:
I ckecked it and the Feature is not deprecated the issue is still open and not resolved
The comment states it "will" be deprecated. But I found no deprecation evidence.